passdmenu.py

```diff
@@ -105,22 +105,40 @@ def get_pass_output(gpg_file, path=PASS, store=STORE):
         sys.exit(1)
     return output.decode('utf-8').split('\n')
 
+def get_user_second_line(pass_output):
+    userline = pass_output[1].split()
+    user = None
+
+    if len(userline) > 1:
+        # assume the first 'word' after some prefix is the username
+        # TODO any better, reasonable assumption for lines
+        # with more 'words'?
+        user = userline[1]
+    elif len(userline) == 1:
+        # assume the user has no 'User: ' prefix or similar
+        user = userline[0]
+
+    return user
```

Comment:
I think it would be fine to drop the second line assumption completely in the future and just have a default regex that strips out a string until a colon/whitespace character or similar.